examples/analytics-tool/README.md

```
@@ -99,7 +99,7 @@ print(f"<IMG>{img_str}</IMG>")
 
 In the end the code prints an encoded image. Inside the tool definition the regex expression is used to extract this string, decode, and plot it.
 
-![](imgs/analytics-output.png)
+![](https://raw.githubusercontent.com/kubernetes-sigs/agent-sandbox/main/examples/analytics-tool/imgs/analytics-output.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)